x/stakeibc/keeper/msg_server_restore_interchain_account.go

```diff
@@ -55,7 +55,7 @@ func (k msgServer) RestoreInterchainAccount(goCtx context.Context, msg *types.Ms
 		TxType:                 icatypes.TxTypeSDKMultiMsg,
 	}))
 
-	if err := k.ICAControllerKeeper.RegisterInterchainAccount(ctx, hostZone.ConnectionId, owner, appVersion); err != nil {
+	if err := k.RegisterInterchainAccount(ctx, hostZone.ConnectionId, owner, appVersion); err != nil {
```

Comment:
Can you please explain this change?

Reply:
as mention above [https://github.com/Stride-Labs/stride/pull/696#discussion_r1156604389](url) we move to the keeper RegisterInterchainAccount that use ICAcontrollerkeeper msgserver to reduce all the duplicated setup we have to make in the function